### PR TITLE
Ground primitives and textured entities on 3D Tiles

### PR DESCRIPTION
**Pull request by lilleyse**
Follows up on https://github.com/AnalyticalGraphicsInc/cesium/pull/7422 and adds support for ground primitives and textured entities on 3D Tiles.

The default classification type is now `ClassificationType.BOTH` everywhere.

on-3d-tiles

[locahost demo](http://localhost:8080/Apps/Sandcastle/index.html#c=vVdtb9s2EP4rXPZBMubI8ov8lhcscQI0QJMWibt9qIuAkmiZjUQKJOXGHfLfdxRFW3KSLgO2Bgli8h7ePffC47nTQWcMkUec5SlBfIkKSVmCMAr7cYYUTSVRSHEUpVhKutwgzLhaEbGVEwB4C7bGAq0p+QaSE8TINzQjkhaZ90e55zpRuZxxpjBlRDht9NeCIfhRRAjY+ij4msZETO3BSBCsyJ9cpPHcQNzWgj21jhZswTrAGjEuMpw2iGgH8hVXPBE4y4gSG8PMihvUzL/+xdwI3YpQIdItiSvObonkhYiItxQ8O5MAvIrdoT8aWDbGa09GhBEvFzSjiq6J9HAcu5XdGuw759mc1wWGYRXfCCvKWcXok0iBseN5Hfi9KzN0gRXuNIh3ZuYkNUc7Hxvud2yGvkrOnKMqco2QFZLEjQxDeveCaLENqrTB9e2hfcXTtoHsKd/kZJuM2TORN7u8u/p0fd+/uJ9fvb+sEvIiP0+qDRT4qyzvtNjyjHjKoRIdkYTY7QVBG/nm1wtazhvS/iKDbeXOIb4SZ+SFNCz5fnjbKCwUEoTB3QCAqfl0Y5A8y7GgkNnqAjLOZv9ncuSKf5uiJYamUEXhNYs/NdyvkTARv8MsjrBUKdHgOU+SlJwXSnHmOnfg0F7AoTMpUZA2WhYs0htutCLRA4lbtmO9Ul9a1QmqwEcG+np4DPqXHfzpZbY8DbGwdKtGiGKSqxV0TqmcGs8twUaskpSHxCtPzOHAWQIKpLKagMLb0Uc2XUxy4JfyZI8S5BD99i/Mt3Z+2/fjmsdEd73KlRWhyUpNkQ/AElfV0gPj0QMvlKcEjh7c7cmdMmVCB6piHhUZYcpLiLpMif54voEm7lQQRx/a14vzPN2cUxbDayh3+ttWb+tFNmDhQyiJWOMQyrt2yjGOOC1PFqGMBA2Ju02ckbWaPgPxmyIL4e2sxFX06RK5VN7gG7u/Pad/BFGFsIl60hTLgtBdG4uyo+crGoFueydru+ULpzdAhJl9oryQF2UU7nJ49okHWYUn27LRmmUhljgiTaWljn6p8hbH8Fm6dQZQPCyhqojhojX34ZqYbd/z61b4cmk62X9qpBnbsm4EqEnLy/qiMcifrjnlGkJt63670eq2eLdlldt4ZroirrES9HFnwKwHpSvzHQO3xqZ2WUyFQyKo2oCO6sKVa1r1xKomcp5uEvBkWi+SFSUCi2i1ge0a6Y8G+85K3R9G+gxu8Mb9fNj1+t3JoD+c9Eajfnfg93o6dcPJeOwPe0Hfn/j6fxsZYH8SBJOx3/MHve5oC5x0g9F4NOr2gsEO2O/2ev4kmPSHFjgCM/3eYOKPh1uFg34w7vUG/iAY+OMtbtSdDIMgGE4m4y+tVnvnegYzpaAwNU7LuYpmONEjVOno/Xue8PuZfoy8r3ni1I49n0rQj8aS8w/zd9srWGtwPNcQCSn7vJ1+H5XmMrPD1zlM19YyQElKIg143uT1j6kAr0qy9wLLk39guesUT7WJfI9T1a1/Eq355e3t2dXNm5j1L1D5ov4kas1xs8bwy9ErL/c1YYVbpV3XwUH74LgcjU7N4d9plnOh9PDlQj0qAkM+lKjshAXMBcqLpLT941f7otXcCeHxS4Ru0FNUTk8DuH32z/fG9mzZCoAVtPEpGuSPte2QCxgtDwXc6kLWhU97ZinLYRKtGV8ToSA66SFOacKmKKNxnJLnFg8Vz6eo17BqRSGHySarS8HscceG6Dima0Tjk8XB3vfHxYG5kyBZFml6R7+TxcHpcQfwjWMpx9rOB+Ca4o2GrLqn782m53nHHVg+P1W5DHDDSUtO35VPhQWX2yYkCkoDDkGnToAEhIHB6rDrwwOml/gRlnYlFcn1Ej7G8FXuMIQJA9ZrnBbwRce8Rm1ULj/lgIBNp7Ti7Mg0rOorofVCAGAV/EhvqcHy/xs)

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by lilleyse** [comment]
@likangning93 can you review the primitive changes and @bagnell the `Scene`/`GlobeDepth` changes?

**Comment by pjcozzi** [comment]
This + #7437 fixes #6636.

**Comment by pjcozzi** [comment]
👍 at quick glance!

File: CHANGES.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+
+* Added support for textured ground entities (entities with unspecified `height`) and `GroundPrimitives` on 3D Tiles.
 
 ##### Fixes :wrench:
-* Fixed an issue where classification primitives with the `CESIUM_3D_TILE` classification type would render on terrain. [#6568](https://github.com/AnalyticalGraphicsInc/cesium/issues/6568)
-* Fixed an issue where 3D Tiles would show through the globe. [#6867](https://github.com/AnalyticalGraphicsInc/cesium/issues/6867)
+* Fixed an issue where classification primitives with the `CESIUM_3D_TILE` classification type would render on terrain.
+* Fixed an issue where 3D Tiles would show through the globe.
+
+##### Breaking Changes :mega:
```
Comment (mramato) [inline CHANGES.md]: Breaking changes should go first, before everything else.

**Comment by lilleyse** [comment]
More CI timeouts. I restarted the build.

**Comment by lilleyse** [comment]
Well the previous commit passes, so I think this is fine. The last commit was CHANGES.md only.

**Comment by bagnell** [comment]
> Well the previous commit passes, so I think this is fine. The last commit was CHANGES.md only.

That's why I merged it.

**Pull request merged**